src/test/java/com/solvd/micro9/users/messaging/UserProducerTest.java

```diff
@@ -0,0 +1,85 @@
+package com.solvd.micro9.users.messaging;
+
+import com.google.gson.Gson;
+import com.solvd.micro9.users.domain.aggregate.User;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.*;
```

Comment:
don't use wildcard import

by the way, checkstyle and many other tools automatically can handle it

Reply:
fixed f6768c5